components/event-publisher-proxy/pkg/handler/nats/handler.go

```diff
@@ -99,8 +99,8 @@ func (h *NatsHandler) ServeHTTP(writer http.ResponseWriter, request *http.Reques
 	return
 }
 
-func (h *NatsHandler) publishLegacyEventsAsCE(writer http.ResponseWriter, request *http.Request) {
-	event := h.LegacyTransformer.TransformsLegacyRequestsToCE(writer, request)
+func (h *Handler) publishLegacyEventsAsCE(writer http.ResponseWriter, request *http.Request) {
+	event := h.LegacyTransformer.TransformLegacyRequestsToCE(writer, request)
```

Comment:
(nit)`Transformer transforms` is grammatically correct. What was your rationale to change it to `transform`?

Reply:
Why introducing a new naming convention? Let's be consistent and use the same naming convention everywhere.

Reply:
`TransformLegacyRequestsToCE` is newly introduced. Why `TransformsLegacyRequestsToCE` was not ok?